x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/use_create_analytics_form.ts

```diff
@@ -94,6 +95,12 @@ export const useCreateAnalyticsForm = (): CreateAnalyticsFormProps => {
       ? jobConfig
       : getJobConfigFromFormState(form);
 
+    if (isAdvancedEditorEnabled) {
+      destinationIndex = Array.isArray(analyticsJobConfig.dest?.index)
+        ? analyticsJobConfig.dest?.index[0]
```

Comment:
wonder if this should be `.join(',')` instead?

Reply:
This is the dest index so shouldn't be multiple indices here.